Testing/SystemTests/tests/framework/GenerateLogbookTest.py

```diff
@@ -37,7 +37,7 @@ def cleanup(self):
         mtd.clear()
 
     def validate(self):
-        self.tolerance = 1e-3
+        self.tolerance = 1e-1
```

Comment:
If the columns are now types, shall we revert the tolernaces?

Reply:
Can do, on Ubuntu the system tests passes locally without any issues.

Reply:
Could you check windows too?

Reply:
I've just tried this locally on my Windows machine and it looks good.

Reply:
I'll approve it.